grabbit/src/main/groovy/com/twcable/grabbit/client/batch/workflows/impl/DefaultWorkFlowManager.groovy

```diff
@@ -27,8 +28,10 @@ import org.apache.felix.scr.annotations.Deactivate
 import org.apache.felix.scr.annotations.Reference
 import org.apache.felix.scr.annotations.Service
 
+import javax.annotation.Nonnull
+import java.lang.String as WorkflowID
 import java.util.concurrent.ConcurrentHashMap
-import java.util.concurrent.atomic.AtomicInteger
+import java.util.concurrent.atomic.AtomicInteger as JobUseCount
```

Comment:
This seems super weird but I understand why this will be useful :smile_cat: from code understanding perspective.